File: Python/1013.py
values = input().split(' ')
a, b, c = values

ab = (int(a)+int(b) + abs(int(a)-int(b)))/2 # 1º
result = (int(c)+int(ab) +abs(int(c) - ab))/2 # 2º

result = int(result)
print(f"{result} eh o maior")

"""
1º  The sum of a and b ensures the final result is at least the larger of the two.
    abs(int(a) - int(b)) calculates the absolute difference between a and b.
    Dividing by 2 serves as a "rounding" mechanism to consider the absolute difference.

2º  Calculates the largest between c and the previous value (ab). The logic resembles line 3.

"""

File: JavaScript/1013.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split(' ');

let A = parseInt(lines[0]);
let B = parseInt(lines[1]);
let C = parseInt(lines[2]);

let testAB = (A+B+Math.abs(A-B))/2;
let maiorABC = (C+testAB+Math.abs(C-testAB))/2;

console.log(`${maiorABC} eh o maior`);